_topic_maps/_topic_map_rosa_hcp.yml

```diff
@@ -167,8 +167,12 @@ Topics:
   File: rosa-sts-aws-prereqs
 - Name: Required IAM roles and resources
   File: rosa-hcp-prepare-iam-roles-resources
-- Name: ROSA with HCP limits and scalability
-  File: rosa-hcp-limits-scalability
+##### NOTE: THE BELOW IS REMOVED AS PART OF OSDOCS-13310
```

Comment:
Where abouts is the preview for this file?

Reply:
It shouldn't appear in the previews because this PR removes it